src/Configuration.php

```diff
@@ -111,6 +116,37 @@ public function parser(): Parser
         return $this->parser;
     }
 
+    public function secureParser(Constraint ...$extraConstraints): ValidateDuringParsing
+    {
+        $constraints           = array_merge($this->validationConstraints, $extraConstraints);
+        $timeVerification      = null;
+        $signatureVerification = null;
+
+        foreach ($constraints as $i => $constraint) {
```

Comment:
This is horrible, honestly but I don't want to run duplicated signature/time verifications